README.rst

```diff
@@ -57,23 +109,36 @@ Installation Requirements
     parameter. (This is required to ensure that all databases have been
     configured with the same version of the replication set configuration.)
 
+You will also need to install ``pgshovel`` in your system. As of this writing it is not available on PyPI, so you will need to download it and run ``python setup.py install``.
+
+All PGShovel administrative actions are performed via the ``pgshovel-admin`` tool, which comes with the ``pgshovel`` package once installed.  Please consult ``pgshovel-admin`` for commands and other options.
+
 Creating a Cluster
 ------------------
 
-.. todo:: Rewrite this as a tutorial using pgbench after the replication worker is done.
-
+To begin, initialize a Cluster.
 ::
 
     pgshovel-admin cluster initialize
 
+Then create a Replication Set (here named "example") based on the configuration file.
+
 ::
 
     pgshovel-admin set create example /path/to/configuration.pgshovel
 
+
+The configuration file is the protobuf text representation of a ``ReplicationSet``_ config object.  Please see the `protobuf message definition`_ and `example set builder tool`_ for more information.
```

Comment:
I probably wouldn't document the example set script in it's current form, personally.